config/intellij-idea-inspection-scope.xml

```diff
@@ -2,5 +2,5 @@
      all excludes should be specified in build configuration -->
 <component name="DependencyValidationManager">
   <scope name="Checkstyle Inspection Scope"
-      pattern="!file[checkstyle]:target//*&amp;&amp;!file:src/test/resources*//**&amp;&amp;!file:src/it/resources*//**&amp;&amp;!file:src/site/resources/js/google-analytics.js&amp;&amp;!file:src/site/resources/styleguides*//**&amp;&amp;!file:config/intellij-idea-inspections.properties&amp;&amp;!file[checkstyle]:.ci-temp//*&amp;&amp;!file[checkstyle]:bin//*&amp;&amp;!file[checkstyle]:.settings//*&amp;&amp;!file:.classpath&amp;&amp;!file:.project&amp;&amp;!file:.circleci/config.yml&amp;&amp;!file:config/projects-to-test/openjdk-projects-to-test-on.config&amp;&amp;!file:config/projects-for-no-exception-javadoc.config&amp;&amp;!file:.ci/pitest-survival-check-xml.groovy"/>
+      pattern="!file[checkstyle]:target//*&amp;&amp;!file:src/test/resources*//**&amp;&amp;!file:src/it/resources*//**&amp;&amp;!file:src/site/resources/js/google-analytics.js&amp;&amp;!file:src/site/resources/styleguides*//**&amp;&amp;!file:config/intellij-idea-inspections.properties&amp;&amp;!file[checkstyle]:.ci-temp//*&amp;&amp;!file[checkstyle]:bin//*&amp;&amp;!file[checkstyle]:.settings//*&amp;&amp;!file:.classpath&amp;&amp;!file:.project&amp;&amp;!file:.circleci/config.yml&amp;&amp;!file:config/projects-to-test/openjdk-projects-to-test-on.config&amp;&amp;!file:config/projects-for-no-exception-javadoc.config&amp;&amp;!file:.ci/pitest-survival-check-xml.groovy&amp;&amp;!file:docs/GRAMMAR_UPDATES.md"/>
```

Comment:
https://app.circleci.com/pipelines/github/checkstyle/checkstyle/27773/workflows/5a9e8777-0325-440e-824e-fc511299ad19/jobs/684276


See above:
> `<!-- ATTENTION: this file is not used by TeamCity,`
     `all excludes should be specified in build configuration -->`

Have we tried excluding at https://github.com/checkstyle/checkstyle/blob/master/config/intellij-idea-inspections.properties ?

Very last resort is to just change 0 to 1 in SH file to ignore this 1 error. This may be an indication we need a suppression system for this script.
https://github.com/checkstyle/checkstyle/blob/master/.ci/idea-inspection.sh#L57

Reply:
>Have we tried excluding at https://github.com/checkstyle/checkstyle/blob/master/config/intellij-idea-inspections.properties ?

No, forgot about this one. Thanks for the suggestion, trying this now!